ai_news_generator/app.py

```
@@ -1,165 +1,329 @@
+        # Create and run the flow
+        flow = NewsGeneratorFlow(
+            topic=topic, 
+            temperature=temperature, 
+            max_sources=max_sources
+        )
+        
+        # Store flow in session state for tracking
+        st.session_state.workflow_state = flow
+        
+        # Run the flow
+        result = flow.kickoff()
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)